### ISSUE TITLE
[FR]Жидкостной вход на 4-ре слота.

### ISSUE THREAD

**Issue by Pilad** [IdealIndustrial/GT5Unofficial#224]
Для того же БХР будет удобно.
https://github.com/GTNewHorizons/GT5-Unofficial/commit/f7c75cbe3675733eca0279f0c7256daf459e109d

**Comment by SapientMC** [comment]
Какое-то лютое переопределение. Может и удобно, но вот как это отображать/биндить/проверять заполение и т.п. - не совсем понятно. Скорее всего будет куча багов и/или тормозов.

**Comment by WiseProjects** [comment]
в гтнх его больше нет, видимо вырезали по какой-то причине

**Comment by Pilad** [comment]
> в гтнх его больше нет, видимо вырезали по какой-то причине

Нет, там и на 4 и на 9 слотов. Как были так и есть. https://github.com/GTNewHorizons/GT5-Unofficial/blob/master/src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_Hatch_MultiInput.java
